.eslintignore

```diff
@@ -13,6 +13,7 @@ packages/blade/tokens.js
 packages/blade/tokens.d.ts
 packages/blade/utils.js
 packages/blade/utils.d.ts
+packages/blade-old/scripts
```

Comment:
Ignoring from eslint will also remove actual errors which are valid and also prettier will stop 😭

Reply:
Will scope it to only the ts files.